README.md

```diff
@@ -4,6 +4,8 @@ This micro-framework supports remote logging via [LogDNA](https://logdna.com) on
 iOS. The framework itself is written in ObjC for easy integration in Swift and
 ObjC apps.
 
+Requires a deployment target of iOS 9 and above.
```

Comment:
There may be other APIs requiring iOS 9 already. I thought we should add this, given we have never tested it below.